### PR TITLE
Try to fix model request subscribeTo: missing unusedSubscriptions

### LINKED ISSUE #4240: Setting 'first speech' throws errors in speaking lists
**Describe the bug**
If the setting 'Show hint »first speech« in the list of speakers management view' is activated, the console throws error messages. Currently I've found no negative impact. Nevertheless it should be fixed.

image

**How to Reproduce**
1. open a meeting
2. open settings > list of speakers > general > activate 'Show hint »first speech« in the list of speakers management view'
3. open agenda > create an item > project it
4. go to Autopilot > open console > Add someone to the list of speakers (users are needed)

**Expected behavior**
The meantioned setting should not throw errors in the list of speakers.

### PR DESCRIPTION
**Pull request by reiterl**
Resolve #4240 

The `subscription.unusedSubscriptions` here can be undefined, now the code can handle it.

### CODE REVIEW

**Review by bastianjoel — CHANGES_REQUESTED** [review]
The error happens because there is a meeting specific subscription is missing a unused subscription. Please fix this instead of just suppressing the error message. 

Also please log a warning to the console if `subscription.unusedSubscription` is undefined at that place.

**Comment by bastianjoel** [comment]
Also PLEASE improve you PR naming. The title does not provide any helpful information. Something like "Fix subscribeTo handle undefined `unusedSubscription`"

**Comment by reiterl** [comment]
@bastianjoel What is this unusedSubscriptions for? In the interface the field is optional.

File: client/src/app/site/services/model-request.service.ts [hunk 1]
```diff
@@ -46,7 +46,12 @@ export class ModelRequestService {
     public async subscribeTo({ modelRequest, subscriptionName, ...config }: SubscribeToConfig): Promise<void> {
         if (this._modelSubscriptionMap.has(subscriptionName)) {
             const subscription = this._modelSubscriptionMap.get(subscriptionName);
-            if (subscription.unusedSubscription.closed) {
+            if (!subscription.unusedSubscription) {
+                console.warn(
+                    `The subscription ${subscriptionName} in the modelSubscriptionMap hasn't got an unusedSubscription.`
+                );
+                subscription.unusedSubscription = this.getUnusedSubscription(config.unusedWhen);
+            } else if (subscription.unusedSubscription?.closed) {
```
Comment (bastianjoel) [inline client/src/app/site/services/model-request.service.ts:54]:
```suggestion
            } else if (subscription.unusedSubscription.closed) {
```

**Review by bastianjoel — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]

**Pull request merged**